modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,165 +8,292 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
-|   |  *URI*   |       *appName*    |  *appURI*
-| 0 | /        | {subdomain}   | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | /v1/applicationX | application-x | /api/ | http
+| 2 | /v1/applicationY | application-y | /api/ | http
+| 3 | /v2/applicationx | application-x-v2 | /api/ | http
 |===
-This rule automatically maps any request passed to a subdomain of `example.com` to the corresponding `appName`:
 
-* Passing `api.example.com` would redirect to `api.cloudhub.io` +
-* Passing `application.example.com` is mapped to `application.cloudhub.io`.
 
-The same applies to the https://en.wikipedia.org/wiki/Subject_Alternative_Name[Subject Alternative Names] (SANs) of your SSL endpoints.
+== Using Patterns in Mapping Rules
+
+A _pattern_ is a string that defines a template for matching an input text.
+The value in curly brackets (`{   }`) is treated as a variable.
 
-Instead of using a wildcard certificate, which permits any subdomain application name to be passed through the dedicated load balancer, you can restrict the allowed subdomain names. To do this, you must have different SANs configured for a certificate's common name. You can use the `subdomain` variable to map the subdomain portion of your domain name to your application.
+* Variable *names* can contain only lowercase letters (`a-z`) and no other characters.
+* Variable *values* can contain the characters `a-z0-1.&?-_`, but can't contain slash characters (`/`).
 
-In this example:
 
-* Two deployed applications:
-** `dev-app`
-** `qa-app`
-* An SSL endpoint with the SANs:
-** `dev.example.com`
-** `qa.example.com`
-* Mapping rule:
-+
-[%header,cols="10a,20a,20a,10a"]
+[IMPORTANT]
+DLBs don't support patterns in the *Output Path* (`appURI`).
+
+If you set up a DNS CNAME record to map `example.com` to `_lb-name_.lb.anypointdns.net`,
+you can then use a mapping rule to map `app.example.com` to a different deployed CloudHub Mule application name.
+
+You can route external requests to `+https://app.example.com+` to `+http://app.cloudhub.io/example+`:
+
+[%header,cols="10a,20a,20a,20a,10a"]
 |===
-|Rule # |Input URL 2+^| Output URL
-|   |  *URI*   |       *appName*    |  *appURI*
-| 0 | /        | {subdomain}-app   | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+|1 | app.example.com | app | /example | http
 |===
 
-This rule maps the subdomain part of your domain name to the application name:
+Alternatively, you can define a pattern to hold the input value:
 
-* `dev.example.com` redirects to `dev-app.cloudhub.io`.
-* `qa.example.com` redirects to `qa-app.cloudhub.io`.
+[%header,cols="10a,20a,20a,20a,10a"]
+|===
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+|1 | example.com/{myPattern} | app-{myPattern} | /data | http
+|===
 
+In this case, both `example.com/bookings` and `example.com/sales` match to `app-bookings/data` and `app-sales/data` respectively, because the variable `_myPattern_` holds these values.
```

Comment:
```suggestion
In this case, both `example.com/bookings` and `example.com/sales` match `app-bookings/data` and `app-sales/data` respectively, because the variable `_myPattern_` holds these values.
```
Had you meant to use "map to" instead of "match to"?